.gitignore

```diff
@@ -133,3 +133,6 @@ data.db
 
 # Jetbrains pyCharm ide
 .idea
+
+# Database migrations
+migrations/versions/*.py
```

Comment:
I don't think we should be gitignoring these files. Otherwise we could end up with devs having different versions of the database.

Reply:
I initially thought contributors would use alembic to manage database migrations, which could create potential merge conflicts as people upgraded, downgraded and made changes, so I added the migrations/versions/*.py to gitignore. But since we're not going to use alembic in the development phase, it makes perfect sense to remove the path from gitignore, so I'll take it out.